cli/root.go

```diff
@@ -981,6 +981,8 @@ func (p *prettyErrorFormatter) format(err error) {
 		msg = sdkError.Message
 		if sdkError.Helper != "" {
 			msg = msg + "\n" + sdkError.Helper
+		} else if sdkError.Detail != "" {
```

Comment:
for reviewers:

Message: "Unable to validate parameter X"
Detail: "parameter value must match one of options"

^ this should be consistent with site UI error